src/dataExplorer/components/Results.tsx

```diff
@@ -1,4 +1,11 @@
-import React, {FC, useState, useContext, useMemo, useCallback} from 'react'
+import React, {
+  FC,
+  useState,
+  useContext,
+  useMemo,
+  useCallback,
+  useEffect,
```

Comment:
Minor feedback that goes along way, but absent any other organizing scheme, I prefer to alphabetize these lists. It's one of those small annoyances at write time that have an outsized impact on ease of reading comprehension

Reply:
I'll do it for this one example here. But note that there are linter rules which can enforce, and auto-fix for us.